dashboards/grafana-dashboard-insights-compliance-general.template.yaml

```diff
@@ -133,7 +133,7 @@ objects:
                 "datasource": {
                   "uid": "$datasource"
                 },
-                "expr": "sum(increase(rails_requests_total{application=\"compliance\"}[$__range]))",
+                "expr": "sum(increase(rails_requests_total{application=\"compliance\"}[$__range])) * $qe + sum(increase(rails_requests_total{application=\"compliance\",org_id!~\"${QE_ACCOUNTS}\"}[$__range])) * (-($qe - 1))",
```

Comment:
The `org_id` matches are lacking exact match.

Reply:
It is intentional, we need the regex as we have to match against multiple org_ids, like `123|456|789` but I could adjust it on the other side as `^123|456|789$` if that is enough...

Reply:
As per Prometheus docs:
> Regex matches are fully anchored. A match of env=~"foo" is treated as env=~"^foo$".

Disregard.